vendor/engines/split_accounts/app/models/split_accounts/reports/export_raw_transformer.rb

```diff
@@ -12,7 +12,9 @@ def transform(original_hash)
       private
 
       def split_percent(order_detail)
-        number_with_precision(order_detail.split.percent, strip_insignificant_zeros: true) + "%" if order_detail.split
+        if order_detail.respond_to?(:split) && order_detail.split
+          number_with_precision(order_detail.split.percent, strip_insignificant_zeros: true) + "%"
+        end
       end
     end
   end
```

Comment:
This looks good.  Or we could extend order_detail to have a `def split; []; end`  Hmm, I'm not sure I'm crazy about that idea.